tensorflow/lite/micro/kernels/ethos-u/ethosu.cc

```diff
@@ -0,0 +1,93 @@
+#include <ethosu_driver.h>
+
+#include "tensorflow/lite/c/common.h"
+#define FLATBUFFERS_LOCALE_INDEPENDENT 0
```

Comment:
Can you add a comment about why this macro is needed?

Reply:
Actually it does not seem be needed. Removing it.